cli/kubectl-hyperkube

```diff
@@ -273,7 +329,8 @@ def get_cluster_status(ctx, environment, status):
             (f'{ctx.obj["url"]}/{ctx.obj["stage"]}'
              f'/clusters/cluster-status?cluster_status={status}'
              f'&environment={environment}'),
-            headers=ctx.obj['headers']
+            headers=ctx.obj['headers'],
```

Comment:
You make the same changes in a lot of places. Maybe a hyper-kube request method should be created and that used?

Reply:
Sounds good. Let me make a change and see what that looks like.